tools/apiview/parsers/rust-api-parser/ci.yml

```diff
@@ -0,0 +1,114 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/apiview/parsers/rust-api-parser
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/apiview/parsers/rust-api-parser
+
+extends:
+  template: /eng/pipelines/templates/stages/1es-redirect.yml
+  parameters:
+    stages:
+      - stage: 'Build'
+        variables:
+          - template: /eng/pipelines/templates/variables/image.yml
+          - name: NodeVersion
+            value: '18.x'
```

Comment:
You should be on 22.

Reply:
again.. copied this from the JS parser, I agree, we should change this.